Rules.py

```diff
@@ -169,7 +193,7 @@ def global_rules(world):
     set_rule(world.get_entrance('Dark Lake Hylia Drop (South)'), lambda state: state.has_Pearl() and state.has('Flippers'))  # ToDo any fake flipper set up?
     set_rule(world.get_entrance('Dark Lake Hylia Ledge Fairy'), lambda state: state.has_Pearl()) # bomb required
     set_rule(world.get_entrance('Dark Lake Hylia Ledge Spike Cave'), lambda state: state.can_lift_rocks() and state.has_Pearl())
-    set_rule(world.get_entrance('Dark Lake Hylia Teleporter'), lambda state: state.has_Pearl() and (state.has('Hammer') or state.can_lift_rocks()))  # Fake Flippers
+    set_rule(world.get_entrance('Dark Lake Hylia Teleporter'), lambda state: state.has('Flippers') and state.has_Pearl() and (state.has('Hammer') or state.can_lift_rocks()))  # Fake Flippers
```

Comment:
Hmm... this is the whirlpool warp from north east dark world to lake hylia. Fake flippers I'm fairly certain fake flippers are possible here (The flippers requirement for glitchless gets added in `no_glitches_rules`.).

However it *is* in the wrong region right now, and if properly put in North East Dark World, this would be simplfied to has_pearl().